builder/bflag.go

```diff
@@ -12,28 +13,32 @@ const (
 	stringType
 )
 
-type BuilderFlags struct {
+// BFlags contains all flags information for the builder
+type BFlags struct {
```

Comment:
I tend to agree with @hqhq - having 'b' or something (like "builder")  in there to make it clear its not the same "flags" as the cmd line stuff is useful.  But no biggie either way.